docs/development/releasing.rst

```diff
@@ -299,21 +299,26 @@ Post-Release procedures
    bugfix or a new major version.  You may also need to update the contributor
    list on the web site if you updated the ``docs/credits.rst`` at the outset.
 
-#. Open a pull request to the astropy *master* branch to
+#. Open a PR to the astropy *master* branch to
    update the ``CHANGES.rst`` to reflect the date of the release you just
    performed and to include the new section of the changelog.  Often the easiest
    way to do this is to use ``git cherry-pick`` the changelog commit just before
    the release commit from above. If you are not sure how to do this, you might
    be better off copying-and-pasting the relevant parts of the maintenance
-   branch's ``CHANGES.rst`` into master.
-
-#. Notify the Continuous Integration maintainer about the new release.
-   While ``conda-forge`` bot automatically opens a PR for stable releases,
-   you still have to notify the Conda Distribution Maintainer about LTS
-   release. Typically, you should wait to make sure
-   ``conda-forge`` and possible ``conda`` works before sending out the public
-   announcement (so that users that want to try out the new version can do
-   so on ``conda``).
+   branch's ``CHANGES.rst`` into master. In the same PR, you also have to
+   update ``docs/whatsnew/index.rst`` and ``docs/whatsnew/X.Y.rst`` to link to
+   "what's new" documentation in the released RTD branch, using the existing
+   text as example.
+
+#. Notify the Conda Distribution and Continuous Integration maintainers
```

Comment:
Oh, I haven't noticed this before. Remove the first sentence, and add a new point after this one: 

* "When the conda-forge packaging is ready, email the Anaconda maintainers about the release(s) so they can update the versions in the default channels."

* "Once the release(s) are available on the default conda channels, prepare the public announcement. For new major release, coordinate with the astropy coordinators, for bugfix release(s) an e-mail to the mailing lists `astropy-dev`` and ``Astropy`` is sufficient."

Reply:
Isn't the second point you mentioned already the last point in this section, with "Send out release announcement emails to email lists..."?

Reply:
yes, that's right. it's just adding a tiny nuance that the announcement is expected to be coordinated, and not sent out by the release manager but by the coordinators.

Reply:
Anyway, I attempted to apply your comments to where I think is appropriate. :woman_shrugging: